app/javascript/react/screens/App/Overview/components/Migrations/MigrationInProgressListItem.js

```diff
@@ -34,13 +37,20 @@ const MigrationInProgressListItem = ({
   setMigrationsFilterAction,
   cancelPlanRequestAction,
   isCancellingPlanRequest,
-  requestsProcessingCancellation
+  requestsProcessingCancellation,
+  loading,
+  toggleScheduleMigrationModal
 }) => {
   const { requestsOfAssociatedPlan, mostRecentRequest } = getRequestsOfPlan({ plan, allRequestsWithTasks });
   const waitingForConversionHost = isWaitingForConversionHost(mostRecentRequest);
 
   const isInitiating = reloadCard || !mostRecentRequest || mostRecentRequest.request_state === 'pending';
 
+  const showScheduleMigrationButton =
+    plan.options.config_info.warm_migration && plan.options.config_info.warm_migration_cutover_datetime === null;
```

Comment:
Didn't notice this before, but I think the `=== null` check here is too strict, since a plan will initially not have a `warm_migration_cutover_datetime` defined. We should probably just have this check for a falsy value.

Reply:
Fixed (I think :smile:).